question-servers/freeform.js

```diff
@@ -165,7 +165,12 @@ module.exports = {
             const resolvedElement = module.exports.resolveElement(elementName, context);
             const cwd = resolvedElement.directory;
             const controller = resolvedElement.controller;
-            const pythonArgs = [elementHtml, data];
+
+            let dataCopy = _.cloneDeep(data);
+            if ('base_url' in data.options) {
```

Comment:
is there ever a case where `base_url` wouldn't be present?

Reply:
I'm not familiar with the specifics of how the PL internals work, but I've found `elementFunction` gets called occasionally with a blank `data.options` so the check is there so nothing weird happens